### PR TITLE
Add simple travis conf

### PR DESCRIPTION
**Pull request by maclav3**

### CODE REVIEW

**Comment by maclav3** [comment]
For now, there are no exceptions defined for PEP8. Knowing PEP8, some will have to be added later, but let's just add them as we go

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+language: python
+python:
+  - "3.5"
+install:
+  - make venv
+script:
+  - make pep8
```
Comment (wojciech-galan) [inline .travis.yml]: + for `pep8`

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,14 @@
+venv: venv/bin/activate
+
+venv/bin/activate: requirements.txt
+	@test -d venv || virtualenv venv
```
Comment (wojciech-galan) [inline Makefile]: Maybe specify the Python version? `virtualenv --python=/usr/bin/python3.5 venv` instead of `virtualenv venv`

Comment (maclav3) [reply Makefile]: good point

**Pull request merged**